calico-cloud/image-assurance/scan-image-registries.mdx

```diff
@@ -134,6 +136,23 @@ You can scan images and report results back to {{prodname}}, or scan images loca
 ./tigera-scanner scan ubuntu:latest --fail_threshold 7.0 --warn_threshold 3.9
 ```
 
+**Scan images with input and output file**
+
+The input file must have the following JSON structure:
+
+```json
+{
+  "images": [
+    "ubuntu:latest",
+    "alpine:latest"
+  ]
+}
+```
+
+```shell
+./tigera-scanner scan --input_file images.json --output_file results.json
+```
```

Comment:
Should we also include an example somewhere to demonstrate scanning multiple images using the command-line arguments method of specifying multiple images?